react/CardGroup/CardGroup.js

```diff
@@ -0,0 +1,15 @@
+import Card from '../Card/Card';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function CardGroup({ children, ...restProps }) {
+  return (
+    <Card {...restProps} group>
+      {children}
```

Comment:
You don't really need to access the `children` here, i would refine this to `<Card group {...props} />` which simplifies the code here, but importantly leaves the prop type validation of `children` in the `<Card>` component itself.